index.html

```diff
@@ -8,6 +8,13 @@
 </head>
 <body>
     
+    <header>
+        <h1>BEST PAGE OF ALL TIME 🥳</h1>
```

Comment:
Sentence casing is preferred because then we can control the style of it in our CSS instead.

Reply:
👍🏽